xbmc/cores/VideoPlayer/Interface/StreamInfo.h

```diff
@@ -30,6 +30,14 @@ enum StreamFlags
   FLAG_STILL_IMAGES = 0x100000
 };
 
+enum class StreamHdrType
+{
+  HDR_TYPE_NONE,
+  HDR_TYPE_DOLBYVISION,
+  HDR_TYPE_HLG,
+  HDR_TYPE_HDR10
```

Comment:
Since HDR10 is the most common and primary supported format maybe worth change order to:

  HDR_TYPE_NONE,
  HDR_TYPE_HDR10,
  HDR_TYPE_DOLBYVISION,
  HDR_TYPE_HLG,

Reply:
Done.